### PR TITLE
Technical Specifications -- combing through and clarifying.

### PR DESCRIPTION
**Pull request by meg-huggingface**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,13 +44,16 @@ Version 1.0 / 23.May.2022
...
 
-1.  Layer normalization applied to word embedding layer
-
-2. [ALiBI positional encodings](https://arxiv.org/pdf/2108.12409.pdf)
+* Decoder-only architecture
+* Layer normalization applied to word embeddings layer (`StableEmbedding`; see [code](https://github.com/facebookresearch/bitsandbytes), [paper](https://arxiv.org/pdf/2110.02861.pdf))
+* ALiBI positional encodings ([paper](https://arxiv.org/pdf/2108.12409.pdf)), with GeLU activation functions
+* 176 billion parameters:
+    * 70 layers, 112 attention heads
+    * Hidden layers are 14336-dimensional
```
Comment (cakiki) [inline README.md:52]: I feel a comma would make the dimensionality easier to read. i.e. `14,336-dimensional`

**Review by cakiki — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (stas00) [reply README.md:52]: these aren't big numbers, and 2,048 suggested below is definitely looking odd.

File: README.md [hunk 2]
```diff
@@ -44,13 +44,16 @@ Version 1.0 / 23.May.2022
...
+    * Sequence length of 2,048 tokens used ([tokenizer code link](https://huggingface.co/bigscience/tokenizer))
```
Comment (stas00) [inline README.md:53]:
```suggestion
    * Sequence length of 2048 tokens used ([tokenizer code link](https://huggingface.co/bigscience/tokenizer))
```

2048 is much more readable than 2,048 to us programmers.

**Review by stas00 — APPROVED** [review]
Thank you for the improvements, @meg-huggingface 

Left one possible suggestion

File: README.md (hunk 1 above)
Comment (meg-huggingface) [reply README.md:52]: Committed change.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (meg-huggingface) [reply README.md]: Done.

**Pull request merged**